### PR TITLE
New package: SigmaNight.BasiliskLLM version 0.1a8

### PR DESCRIPTION
**Pull request by clementb49**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/186062)

### CODE REVIEW

**Comment by clementb49** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\basilisk_llm\basilisk.exe returned exit code: -1073741515
> Executable C:\Users\validator\AppData\Local\Programs\basilisk_llm\basilisk.exe returned exit code: -1073741515
> Executable C:\Users\Validator\AppData\Local\Programs\basilisk_llm\basilisk.exe returned exit code: -1073741515

Automated error analysis suggests -1073741515 may mean STATUS_DLL_NOT_FOUND - possibly due to a dependency on `Microsoft.VCRedist.2015+.x64` (or `.x86`).

(Automated response - build 897.)

File: manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: SigmaNight.basiliskLLM
+PackageVersion: 0.1a8
+InstallerType: inno
+Installers:
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x64.exe
+  Architecture: x64
+  InstallerSha256: 373B54D72F2C12897647A3E3EA57BEEC43B232F4941161D47E677BB601228B16
```
Comment (stephengillie) [inline manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml:10]:
```suggestion
  InstallerSha256: 373B54D72F2C12897647A3E3EA57BEEC43B232F4941161D47E677BB601228B16
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

File: manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  Scope: machine
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x64.exe
+  Architecture: x64
+  InstallerSha256: 373B54D72F2C12897647A3E3EA57BEEC43B232F4941161D47E677BB601228B16
```
Comment (stephengillie) [inline manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml:16]:
```suggestion
  InstallerSha256: 373B54D72F2C12897647A3E3EA57BEEC43B232F4941161D47E677BB601228B16
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

File: manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x86.exe
+  Architecture: x86
+  InstallerSha256: FD45B800ECC207C689AA569CF9592C0B402798670A4131F428CAA5ABB49DC760
+  Scope: machine
+  InstallerSwitches:
+    Custom: /ALLUSERS
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x86.exe
+  Architecture: x86
+  InstallerSha256: FD45B800ECC207C689AA569CF9592C0B402798670A4131F428CAA5ABB49DC760
```
Comment (stephengillie) [inline manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml:29]:
```suggestion
  InstallerSha256: FD45B800ECC207C689AA569CF9592C0B402798670A4131F428CAA5ABB49DC760
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

File: manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x64.exe
+  Architecture: x64
+  InstallerSha256: 373B54D72F2C12897647A3E3EA57BEEC43B232F4941161D47E677BB601228B16
+  Scope: user
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+
+- InstallerUrl: https://github.com/SigmaNight/basiliskLLM/releases/download/v0.1a8/setup_basiliskLLM_0.1a8_x86.exe
+  Architecture: x86
+  InstallerSha256: FD45B800ECC207C689AA569CF9592C0B402798670A4131F428CAA5ABB49DC760
```
Comment (stephengillie) [inline manifests/s/SigmaNight/basiliskLLM/0.1a8/SigmaNight.basiliskLLM.installer.yaml:23]:
```suggestion
  InstallerSha256: FD45B800ECC207C689AA569CF9592C0B402798670A4131F428CAA5ABB49DC760
  Dependencies:
    PackageDependencies:
      - PackageIdentifier: Microsoft.VCRedist.2015+.x86
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @clementb49,

This package appears to have a dependency on `Microsoft.VCRedist.2015+.x64`:
image

**Comment by clementb49** [comment]
Hello @stephengillie I don't understand why the validation phase isn't passing.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**